holidays/locale/th/LC_MESSAGES/TW.po

```diff
@@ -68,6 +69,10 @@ msgstr "วันไหว้พระจันทร์"
 msgid "國慶日"
 msgstr "วันชาติสาธารณรัฐจีน(ไต้หวัน)"
 
+#. Taiwan Restoration and Guningtou Victory Memorial Day.
+msgid "臺灣光復暨金門古寧頭大捷紀念日"
+msgstr "การฟื้นฟูไต้หวันและวันรำลึกชัยชนะของจินเหมิน กุนหนิงโถว"
```

Comment:
Based on official transliteration here for Kuningtou/Guningtou https://today.line.me/th/v3/article/PGPaaPM
```suggestion
msgstr "วันรำลึกการทวงคืนไต้หวันและชัยชนะยุทธการกู่หนิงโถว"
```

Aside from this, LGTM 👍